### ISSUE TITLE
Location search doesn't work on Android 4.1 emulator

### ISSUE THREAD

**Issue by lemonboston** [schedjoules/android-event-discovery-sdk#106]
Suggestions don't come up on the location picker screen.
Not sure what's the reason because on other, later version emulators it works.

**Comment by dmfs** [comment]
Is this a live-bug?

**Comment by lemonboston** [comment]
Not sure, I don't even know if this is a real bug or only on emulator, just wanted to create the bug so we can check.

**Comment by dmfs** [comment]
Maybe your emulator just doesn't have the play services installed?

**Comment by lemonboston** [comment]
I've looked into this, Play Services was installed on the emulator but with older version (9.2) then we use (9.6), so a `SERVICE_VERSION_UPDATE_REQUIRED` error occurs which is not handled currently (will be part of https://github.com/schedjoules/backlog/issues/118, in this case prompting the user to update).

I've spent about 45 minutes trying to make/change an API 16 emulator to have newer Play Services version, but I couldn't, so I leave it at this now.
Live devices should have the newer Play Services, because according to Play Services release notes
https://developers.google.com/android/guides/releases
they will start to only deprecate Android 2.3.x in the next release, so our minimum Android 4.0+ should be okay.

I created a technical story nonetheless for us to ensure we can test all the supported versions:
https://github.com/schedjoules/backlog/issues/129

@dmfs I think we can close this bug.
